share/picongpu/tests/compileParticlePusher/include/picongpu/param/isaac.param

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2016-2020 Alexander Matthes
```

Comment:
not needed

Reply:
Same happened here. Thanks for pointing out!